tests/cli/ConfigParamsCest.php

```diff
@@ -6,17 +6,27 @@
 
 final class ConfigParamsCest
 {
-    private function moveToTestDir(CliGuy $I): void
+    /**
+     * @beforeClass
+     */
+    public static function _beforeClass(CliGuy $I)
```

Comment:
the name of these functions could be anything as long as they include the `@beforeClass` annotation, right?
why are they `static`?

Reply:
PHPUnit convention - only static methods can be executed before and after class.

We would have to replace `TestUtil::getHookMethods($testClass);` with our own code, which is actually a good idea.

Reply:
I will try to replace TestUtil with Codeception annotation parsing code.

Reply:
instance methods are supported now